chain/network/src/peer_manager/tests/snapshot_hosts.rs

```diff
@@ -186,10 +186,25 @@ async fn invalid_signature_not_broadcast() {
         .send(PeerMessage::SyncSnapshotHosts(SyncSnapshotHosts { hosts: vec![info2.clone()] }))
         .await;
 
-    tracing::info!(target:"test", "Make sure that only the valid message is broadcast.");
+    tracing::info!(target:"test", "Make sure that only the valid messages are broadcast.");
+
+    loop {
```

Comment:
I can't use this function because `peer2` is not a `PeerManager`, it's a `PeerHandle`.

Maybe I could implement something similar for `PeerHandles` :eyes: